infra/debian/compiler/rules

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/make -f
 export DH_VERBOSE = 1
 export NNAS_BUILD_PREFIX = build
-export PRESET = 20230907
+export PRESET = 20240812
```

Comment:
plz split `infra/*` to separate PRs so that `compiler/*` is not mixed in same PR